src/designGrid.css

```diff
@@ -0,0 +1,63 @@
+@import "./variables.css";
```

Comment:
oh got it, it's a pity, it would have been nice to have 😆 

Then yeah remove it and  then it's ready to merge, approving the PR now